pom.xml

```diff
@@ -88,13 +90,13 @@
     <dependency>
       <groupId>com.fasterxml.jackson.core</groupId>
       <artifactId>jackson-databind</artifactId>
-      <version>${jackson-databind.version}</version>
+      <version>${jackson.version}</version>
     </dependency>
 
     <dependency>
       <groupId>com.fasterxml.jackson.datatype</groupId>
-      <artifactId>jackson-datatype-jsr310</artifactId>
-      <version>${jackson.version}</version>
+      <artifactId>jackson-datatype-jdk8</artifactId>
+      <version>${jackson-datetype-jdk8.version}</version>
```

Comment:
From my reading of https://github.com/FasterXML/jackson-modules-java8#usage, it looks like `jackson-datatype-jsr310` and `jackson-datatype-jdk8` are not the same thing. Did you mean to add `jackson-datatype-jdk8` and leave `jackson-datatype-jsr310` in place? Are the types from `jackson-datatype-jsr310` in Jackson core by default in 2.9.x?

Also, it looks like there is a 2.9.6 release of both of those artifacts, so why do we need an explicit version? (http://central.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.9.6/ and http://central.maven.org/maven2/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.9.6/)

Reply:
https://github.com/FasterXML/jackson-datatype-jsr310 says it was renamed, but then their docs in https://github.com/FasterXML/jackson-modules-java8 say to include all three modules.

Reply:
https://github.com/FasterXML/jackson-modules-java8/tree/master/datatypes

The java8 pom can't be used as it's just a parent pom. We'd need to include both jdk8 and jsr310 (and optionally the parameter names one; I like that one myself).

Reply:
My understanding is that the classes in jackson-datatype-jdk8 were not included before this change, so we don't _need_ to include that artifact unless someone wants to use those classes in new code (which seems reasonable).

Reply:
I was reading on the old repo, so my bad 😅 Hence the odd version